free-2.md

```diff
@@ -11,34 +11,41 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Many college students struggle to organize multiple homework assignments, classtimes, meetings, big projects with smaller parts, reoccuring due dates, a calendar for events, and a list of all the little things they need to do "at some point" during the week. They use checklists, calendars, todo lists, sticky notes and other resources, but there doesn't seem to be one resources that can take care of all of those needs in one place. I would like to build a DSL that would allow them to merge all of these things they need to do into one calendar, this would decrease stress and help them to more efficiently complete everything that they needed to do.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would require the students to invest time in setting up their calendars but it would give them the flexibility, ease, and detailed control needed to put in all their due dates at the beginnning of the semester and then add additional pieces as they go. They could more easily express their busy lives in a calendar using this DSL than in the current, widely used software. A DSL is the appropriate tool, because this is a very limited, specific domain, but we want users to be able to easily express anything within it.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+This seems so relevant to the lives of all college students. I have often felt frustrated with the difficulty of specifying exactly what I want on a calendar or of needing to use multiple tools to keep track of everything I am doing throughout the day. I was thinking about it some before I looked at any examples of past projects and was encouraged in my idea when I read about what NCarter3 did for his DSL in 2014 and what aputman did for his project in 2015.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Organizing tasks and activities together.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I would like to allow the user to type a command perhaps in terminal adding a new event or task and for them to immediantly see the updated result in their calendar. This would allow them to easily add new things to their calendar and if I create a simple documentation of what commands are available they can look up what kind of thing they want to add, type it in and then instantly see their result.
 
```

Comment:
Would the calendar be displayed through terminal as well or are you thinking of another type of interface for viewing it?